### DISCUSSION TITLE
Canonical branch for the python-quickstart docs?

### DISCUSSION THREAD

**Discussion by vishakh** [NillionNetwork/discussions#86] [Q&A] [answered]
Hello, https://docs.nillion.com/python-quickstart initially points to the `main` branch..

![image](https://github.com/user-attachments/assets/ac150251-7d72-4ab4-8413-74de281de682)

..but further down in the page the documentation seems to refer to the `update-quickstart-july-release` branch. Which one is currently considered the canonical branch?

**Comment by oceans404** [comment, marked as answer]
Good catch! This should point to the main branch. I just updated the Python Quickstart docs to reflect this

**Reply by vishakh** [reply]
FYI, the docs and the repo are still out of sync. 

![image](https://github.com/user-attachments/assets/592bb9cf-8611-436a-8146-48bb216bcfac)


![image](https://github.com/user-attachments/assets/4f9c527e-7406-4ec5-9897-6ecaed478b9a)

**Reply by oceans404** [reply]
Thanks for pointing that out! I just updated this for a full fix: https://docs.nillion.com/python-quickstart#using-the-python-client-to-run-your-program

Now
- the quickstart client code points to the run_my_first_program.py file
- the quickstart complete branch is main

**Reply by vishakh** [reply]
Awesome! :clap: :clap:
